src/logstash/pipeline/sync_production_locations.conf

```diff
@@ -87,11 +87,11 @@ filter {
   mutate {
     copy => { "os_id" => "[@metadata][_id]" }
     remove_field => [
-      "@version", 
+      "@version",
       "@timestamp",
-      "name_local_value",
+      "local_name_value",
       "description_value",
-      "url_value",
+      "business_url",
```

Comment:
Could you please use `business_url_value` as in other fields?

Reply:
done